integration/single-node-slug-deploy/check.go

```diff
@@ -300,7 +301,6 @@ func postHelloManifest(dir string) error {
 	manifest := &pods.Manifest{}
 	manifest.Id = "hello"
 	manifest.StatusPort = 43770
-	manifest.StatusHTTP = true
```

Comment:
So how was this passing before? Hello should have been listening on HTTPs before?